core/block_render_svg_vertical.js

```diff
@@ -631,6 +631,14 @@ Blockly.BlockSvg.prototype.render = function(opt_bubble) {
   // If there are no icons, cursorX will be 0, otherwise it will be the
   // width that the first label needs to move over by.
 
+  // If the first field is an image, as in extension blocks, and this is a
+  // reporter block, add a horizontal offset.
+  if ((this.inputList[0].fieldRow[0] instanceof Blockly.FieldImage) &&
+    this.outputConnection) {
```

Comment:
Indent continuation lines +4 spaces, here and everywhere: https://google.github.io/styleguide/jsguide.html#formatting-indent

Reply:
all set